modulemd/v1/modulemd-modulestream.c

```diff
@@ -1798,11 +1798,11 @@ modulemd_modulestream_get_nsvc (ModulemdModuleStream *self)
   if (context)
     {
       nsvc = g_strdup_printf (
-        "%s:%s:%" PRIx64 ":%s", name, stream, version, context);
+        "%s:%s:%" PRIu64 ":%s", name, stream, version, context);
```

Comment:
Or use G_INT64_FORMAT

Reply:
Didn’t know about that one, but let’s not change it here. I’ll prep a further patch to change all the usage in libmodulemd later.